Web/starter_project_iconics/components/layout/Header.tsx

```diff
@@ -0,0 +1,118 @@
+import Image from 'next/image'
+import { useState } from 'react'
+import { MdOutlineClose, MdViewHeadline } from 'react-icons/md'
+import logo from '../../public/a2sv-logo.png'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+const Header = () => {
+  const [navigation, setNavigation] = useState(() => [
+    { id: 0, name: 'Home', to: '/', current: true },
+    { id: 1, name: 'Teams', to: '/team/teams', current: false },
+    {
+      id: 2,
+      name: 'Success Stories',
+      to: '/Success_story/success_stories',
+      current: false,
+    },
+    { id: 3, name: 'About Us', to: '/about/about', current: false },
+    { id: 4, name: 'Blogs', to: '/blog/blogs', current: false },
+    {
+      id: 5,
+      name: 'Get Involved',
+      to: '/get_involved/get_involved',
+      current: false,
+    },
+  ])
+
+  const [showNav, setShowNav] = useState(false)
+
+  const handleNavClick = (e: React.FormEvent) => {
+    let id = parseInt((e.target as HTMLAnchorElement).id)
+
+    setNavigation(
+      navigation.map((item) => {
+        if (item.id == id) {
+          return { ...item, current: true }
+        } else {
+          return { ...item, current: false }
+        }
+      })
+    )
+  }
+
+  return (
+    <div className="bg-white flex justify-between p-5">
+      <div className="w-28 lg:w-52">
+        <Image
+          src={logo}
+          alt="a2sv-logo"
+          width={200}
+          height={50}
+          className="object-cover"
+        />
+      </div>
+
+      <div className="w-3/4 flex flex-col gap-y-4  md:ml-0 md:flex-row justify-between">
+        {showNav ? (
```

Comment:
not sure i understand how the showNav works
I am hoping it will make better sense with the screenshots

Reply:
I think you should attach screenshot of both web and mobile view